### PR TITLE
Upgrade to buildroot 2015.02 and remove nginx

### PR DESCRIPTION
**Pull request by andyshinn**
* Upgrades to buildroot 2015.02. Lots of fixes (the current version didn't even build for me).
* Removes the nginx package. Looks like nginx was added in 2014.11 per changelog at http://git.buildroot.net/buildroot/plain/CHANGES?id=2014.11-rc1.
* Changes base to `debian` to save a little space. Why not `ubuntu-debootstrap`? I don't know...

Before merging:

- [x] Someone other than me take a peek at the auto-updated `example/config`.
- [x] I'll test the builder on EC2 with my latest edits to make sure it actually works.

### CODE REVIEW

**Comment by progrium** [comment]
example/config looks fine. mostly added commented lines.

File: example/config [hunk 1]
```diff
@@ -31,35 +32,22 @@ BR2_x86_64=y
 # BR2_xtensa is not set
 BR2_ARCH="x86_64"
 BR2_ENDIAN="LITTLE"
-BR2_GCC_TARGET_TUNE="generic"
-BR2_x86_generic=y
```
Comment (andyshinn) [inline example/config]: `BR2_x86_generic` was a deprecated option and `BR2_GCC_TARGET_TUNE` was changed to be `nocona` (there is no more `generic`). I think these were the two most glaring changes the **could** possibly affect a BusyBox rebuild.

Comment (progrium) [inline example/config]: Yeah, saw those. Should be fine.

On Mon, Mar 16, 2015 at 7:52 PM, Andy Shinn <notifications@github.com>
wrote:

> In example/config
> <https://github.com/progrium/rootbuilder/pull/4#discussion_r26541670>:
>
> > @@ -31,35 +32,22 @@ BR2_x86_64=y
> >  # BR2_xtensa is not set
> >  BR2_ARCH="x86_64"
> >  BR2_ENDIAN="LITTLE"
> > -BR2_GCC_TARGET_TUNE="generic"
> > -BR2_x86_generic=y
>
> BR2_x86_generic was a deprecated option and BR2_GCC_TARGET_TUNE was
> changed to be nocona (there is no more generic). I think these were the
> two most glaring changes the *could* possibly affect a BusyBox rebuild.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/progrium/rootbuilder/pull/4/files#r26541670>.
>



-- 
Jeff Lindsay
http://progrium.com

**Comment by andyshinn** [comment]
Tested locally and was able to build a Docker image with the resulting `rootfs.tar`.

**Pull request merged**